src/server/templates/model.html

```diff
@@ -53,6 +53,14 @@
 
       {% include "lines.html" %}
 
+      <p>
+        At the peak of the epidemic, under this scenario, there will be
+        requiring <span id="selected-region-beds"></span> hospital beds and
+        <span id="selected-region-active-infections"></span> active infections,
+        <span id="selected-region-icu-units"></span> ICU units.
+        Overall there will be <span id="selected-region-infections"></span> infections,
+        which can lead do <span id="selected-region-deaths"></span> deaths.
+      </p>
```

Comment:
I would prefer a list (with commenting paragraph), e.g.:

Under this scenario, with high confidence (90%) we estimate:
* The **maximum number of simultaneously infected people** will be **X-Y** (A-B%)
* The **total number of people infected** will be **X-Y** (A-B%)
* St. similar for beds a daths?

Reply:
That sounds better. Added.